translation/de.json

```diff
@@ -2,162 +2,162 @@
   "[symbol] feed is delayed by [delay] minutes": "[symbol] Feed ist [delay] Minuten verspätet",
   "[symbol] market is now opened.": "[symbol] Markt hat jetzt geöffnet.",
   "[symbol] market is presently closed.": "[symbol] Markt ist derzeit geschlossen.",
-  "A technical momentum indicator that compares a security's closing price to its price range over a given time period. The oscillator's sensitivity to market movements can be reduced by adjusting the time period or by taking a moving average of the result.": "",
+  "A technical momentum indicator that compares a security's closing price to its price range over a given time period. The oscillator's sensitivity to market movements can be reduced by adjusting the time period or by taking a moving average of the result.": "Ein technischer Momentumindikator, der den Schlusskurs eines Wertpapiers mit seiner Preisspanne über einen gegeben Zeitraum vergleicht. Die Sensibilität des Oszillators zu Marktbewegungen kann reduziert werden, indem der Zeitraum angepasst wird, oder indem ein Ergebnis des gleitenden Durchschnitts genommen wird.",
```

Comment:
@matin-deriv did all these `.json` files update when you ran `npm run translations`? or did you do the changes manually?

Reply:
These are the new JSON files added by translations team to Crowdin and I downloaded them along with Korean JSON file using `npm run translations:download`